packages/api/src/schema/schema.yaml

```diff
@@ -906,6 +906,60 @@ components:
           description: Object store ID where the asset is stored
           writeOnly: true
           example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        storage:
+          additionalProperties: false
+          properties:
+            ipfs:
+              $ref: "#/components/schemas/export-task-params/oneOf/1/properties/ipfs"
+        status:
+          type: object
+          additionalProperties: false
+          readOnly: true
+          required: [phase, updatedAt]
+          description: Status of the asset
+          properties:
+            phase:
+              type: string
+              description: Phase of the asset
+              enum:
+                - waiting
+                - ready
+                - failed
+            updatedAt:
+              type: number
+              description:
+                Timestamp (in milliseconds) at which the asset was last updated
+              example: 1587667174725
+            errorMessage:
+              type: string
+              description: Error message if the asset creation failed.
+            storage:
+              type: object
+              additionalProperties: false
+              properties:
+                ipfs:
+                  additionalProperties: false
+                  required: [taskIds]
+                  properties:
+                    taskIds:
+                      type: object
+                      additionalProperties: false
+                      properties:
+                        pending:
+                          type: string
+                          description:
```

Comment:
Does this get cleared by `task-runner` on export completion? I cannot find any reference to it